specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+              },
+              "hostId": {
+                "type": "string",
+                "description": "The network ID of the node."
+              },
+              "rack": {
+                "type": "string",
+                "description": "The rack this node is part of."
+              }
+            }
+          }
+        }
+      }
+    },
+    "ListBackups": {
+      "description": "List of restorable backups for a Cassandra cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of backups.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "Unique resource identifier for this backup, of the form /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/backups/{backupId}. The backupId is the timestamp of when the backup was taken, in number of seconds since the epoch."
+              },
+              "timestamp": {
+                "type": "string",
+                "format": "date-time",
+                "description": "The time this backup was taken, formatted like 2021-01-21T"
+              }
+            }
+          }
+        }
+      }
+    },
+    "ListDataCenters": {
+      "description": "List of managed Cassandra data centers and their properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of data centers.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/DataCenterResource"
+          }
+        }
+      }
+    },
+    "SeedNode": {
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "IP address of this seed node.",
+          "type": "string"
+        }
+      }
+    },
+    "Certificate": {
+      "type": "object",
+      "properties": {
+        "pem": {
+          "description": "PEM formatted public key.",
+          "type": "string"
+        }
+      }
+    },
+    "Tags": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      },
+      "description": "Tags are a list of key-value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters. For example, the default experience for a template type is set with \"defaultExperience\": \"Cassandra\". Current \"defaultExperience\" values also include \"Table\", \"Graph\", \"DocumentDB\", and \"MongoDB\"."
+    },
```

Comment:
Is this the Tags defined by ARM or you created a new property called Tags for your scenario. If it is the former then we can resuse object definition from cosmosdb.json, if latter then maybe rename it to avoid confusion.

Reply:
Tags defined by ARM. I've switch to using the ARMResource properties from cosmos-db.json.